modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/configuration/FFManagementToolbarConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.document.library.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Carolina Alonso
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.document.library.web.internal.configuration.FFManagementToolbarConfiguration"
+)
+public interface FFManagementToolbarConfiguration {
+
+	@Meta.AD(deflt = "false", required = false)
+	public boolean showDesignImprovements();
```

Comment:
@tinycarol Sorry I didn't see this in the previous review, but I believe the naming convention for feature flag methods is `enableWhatever`; in this case it would be something like `enableDesignImprovements`.